Flask-Integration/get-localhost.py

```diff
@@ -0,0 +1,20 @@
+from flask import Flask, jsonify, request
+from spellCheck import *
+
+#export FLASK_APP=test1.py
```

Comment:
Can we change the name of the files in this command?

Reply:
Done, changed to file name